.eslintrc.yml

```
@@ -43,13 +47,13 @@ settings:
     version: 16.3.2
 
 overrides:
-  - files: '*.mdx'
+  - files: '*.{md,mdx}'
     extends:
       - plugin:mdx/recommended
       - plugin:mdx/overrides
     rules:
       import/no-extraneous-dependencies: 0
-      prettier/prettier: 0
+      prettier/prettier: 2
```

```suggestion
      prettier/prettier: error
```

The original value is `0`, so I changed it to `2` without a second thought. 😂